File: Day3/Python/main.py

import sys
import rucksack


def main() -> None:
  ruck = rucksack.Ruck(sys.argv[1])
  print(f"[Part 1] Answer : {ruck.part1()}")
  print(f"[Part 2] Answer : {ruck.part2()}")


if __name__ == "__main__":
  main()

File: Day3/Julia/main.jl
include("./rucksack.jl")

using Printf
using .Rucksack

function main()
    lines = Rucksack.get_file( ARGS[1] )
    @printf("[Part 1] Total score : %d\n", Rucksack.part_1(lines) )
    @printf("[Part 2] Total score : %d\n", Rucksack.part_2(lines) )
end

main()